cloud.profiles.d/edx_sandbox.conf

```diff
@@ -0,0 +1,31 @@
+# This profile is for instances launched for the purpose of
+# building edX sandbox AMIs.
+
+edx_sandbox_ami:
+  provider: mitx
+  size: m3.medium
+  image: ami-427a392a
+  ssh_username: ubuntu
+  ssh_interface: public_ips
+  securitygroupid: sg-1d437d65
+  subnetid: subnet-8671faac
+  block_device_mapping:
+    - DeviceName: /dev/xvda
+      Ebs.VolumeSize: 50
```

Comment:
This is the size we've been using for sandboxes for the past 8 months or so. At least from what the docs say, it doesn't look like we can specify a different size from the AMI when using the boto_ec2.run module.

```
salt.modules.boto_ec2.run(image_id, name=None, tags=None, instance_type='m1.small', key_name=None, security_groups=None, user_data=None, placement=None, region=None, key=None, keyid=None, profile=None)
```